src/python/pants/backend/python/dependency_inference/parse_python_dependencies.py

```diff
@@ -61,23 +87,38 @@ async def parse_python_imports(request: ParsePythonImportsRequest) -> ParsedPyth
         Process(
             argv=[
                 python_interpreter.path,
-                "./__parse_python_imports.py",
+                "./__parse_python_dependencies.py",
                 file,
             ],
             input_digest=input_digest,
-            description=f"Determine Python imports for {request.source.address}",
+            description=f"Determine Python dependencies for {request.source.address}",
             env={
                 "STRING_IMPORTS": "y" if request.string_imports else "n",
-                "MIN_DOTS": str(request.string_imports_min_dots),
+                "STRING_IMPORT_MIN_DOTS": str(request.string_imports_min_dots),
+                "ASSETS": "y" if request.assets else "n",
+                "ASSET_MIN_SLASHES": str(request.asset_min_slashes),
             },
             level=LogLevel.DEBUG,
         ),
     )
     # See above for where we explicitly encoded as utf8. Even though utf8 is the
     # default for decode(), we make that explicit here for emphasis.
     process_output = process_result.stdout.decode("utf8") or "{}"
-    return ParsedPythonImports(
-        (imp, ParsedPythonImportInfo(**info)) for imp, info in json.loads(process_output).items()
+    output = json.loads(process_output)
+
+    return ParsedPythonDependencies(
+        imports=ParsedPythonImports(
+            (key, ParsedPythonImportInfo(**val)) for key, val in output.get("imports", {}).items()
+        ),
+        assets=ParsedPythonAssets(
+            [
+                (
+                    _filepath_to_modname(request.source.file_path) if pkgname is None else pkgname,
```

Comment:
`_filepath_to_modname` gets called repeatedly with the same value. Probably better to save as a variable.